### ISSUE TITLE
Improvements to Code Node Explained Page

### ISSUE THREAD

**Issue by iskerrett** [flowforge/website#985]
### Description

I really like the Core Nodes pages. I would like to suggest some improvements:
1. For the Left hand navigation it would be nice to have this list to be expandable. At the top level should be: Common, Function, Network, Parsers, Sequence, Storage.  Each one of these the user should be able to expand to see the next level.
2. I would like to use the title 'Node-RED Core Nodes Explained'
3. For the introductory text, it would be nice to for each section description include a link to the section in the left hand side and expand it out.
4. I would move the search bar to the top of the left hand side.

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by joepavitt** [comment]
> I would move the search bar to the top of the left hand side.

Just a note to say, the Core Nodes currently use the same templating as the Handbook & Docs, so if we move the search bar to top-left for Core Nodes, it would also move on the Handbook and Docs too. This was top-left originally, but believe @ZJvandeWeg moved it fairly recently.

**Comment by iskerrett** [comment]
If we are using the same template as the documentation and handbook, then can't we just just the expandable format we use in the documentation and handbook?

**Comment by joepavitt** [comment]
yep - that was my explanation here: https://github.com/flowforge/website/issues/985#issuecomment-1645152945
